generators/app/index.js

```diff
@@ -344,6 +351,26 @@ module.exports = class extends BaseGenerator {
                         }
                     }
                 }
+
+                if (!this.options['skip-git']) {
+                    this.isGitInstalled((code) => {
+                        if (code === 0) {
+                            this.gitExec('rev-parse --is-inside-work-tree', (err, gitDir) => {
+                                if (!gitDir) {
+                                    this.gitExec('init', () => {
+                                        this.log('git init successful');
+                                        this.gitExec(['add -A'], () => {
```

Comment:
no need of array for this syntax, remove array brackets here and in next command